recipe/meta.yaml

```diff
@@ -15,6 +15,9 @@ build:
   number: 2
```

Comment:
I think this should be incremented

Reply:
I don't think so actually. I used this autogenerated [PR in the pandas-feedstock](https://github.com/conda-forge/pandas-feedstock/pull/97/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a) as reference, and the build number was not incremented in that. I'm guessing it would have been incremented there if this was necessary. So I assumed that when adding new builds, the existing packages don't get rebuilt, and therefore there's no reason to increment the build number. I'm not totally confident on that, though.